app/views/messages/github_no_deployment_status.text.erb

```diff
@@ -0,0 +1 @@
+:sadparrot: <@<%= @account.id %>>, The Github Deployment <<%= @github_deployment.id %>> of <%= @github_deployment.repository %>@<<%= @github_deployment.sha %>|<%= @github_deployment.ref %>> to *<%= @github_deployment.environment %>* did _not_ start. Please review this repo's `.slashdeploy.yml` file and/or have @OpsEng review this repo's Empire Github App Integration Webhooks (for more details, visit: https://remind.quip.com/UbN0AaMDsn8N#dcPACAHqhCJ)
```

Comment:
SlashDeploy is open source, so messages with content pointing to internal Remind docs is a no go.

Let's update `app/views/documentation/index.html.erb` with some debugging information about the common cases that can result in this scenario, and then change this to link to https://slashdeploy.io/docs:

1. Repository doesn't have a webhook handling the `deployment` event. Or..
2. The service handling the `deployment` event isn't updating GitHub with Deployment Statuses.
   * Service may be hung/down/etc.
   * Service may have ignored the deployment request, if it's filtering for a specific environment (e.g. it'll deploy to `production`, but that provided environment in the `deployment` event was `prod`).